library/axi_dmac/tb/run_tb.sh

```diff
@@ -10,12 +11,21 @@ if [[ ${SIMULATOR} != "modelsim" ]]; then
   cd vcd
   ../run/run_${NAME}
 
-else
+elif [[ ${SIMULATOR} == "modelsim" ]]; then
 
   # ModelSim flow
   vlib work
 
   vlog ${SOURCE} || exit 1
   vsim "dmac_"${NAME} -do "add log /* -r; run -a" -gui || exit 1
 
+elif [[ ${SIMULATOR} == "xsim" ]]; then
+
+  # xsim flow
+
+  xvlog -log ${NAME}_xvlog.log --sourcelibdir . ${SOURCE}
+  xelab -log ${NAME}_xelab.log -debug all dmac_${NAME}
+  xsim work.dmac_${NAME} -gui || exit 1
```

Comment:
Don't we need a 'run -a' equivalent command here ? How does xsim will know how long to run assuming the required execution time is longer than the default value xsim has?

Reply:
Currently, the simulation will not start after the GUI opens. It have to start it manually. Will look into this, maybe I'll find the right option/flag to run the simulation automatically.

Reply:
On xsim the only run flag is a -R which does a "run all;quit", so the user have to open up the GUI manually, if he/she wants to do some debug or see some waves. (run a "xsim work.dmac_<test_name> -gui" command)

The xsim will run until reaches $finish (each test bench should have one), we can specify a max delta id, to prevent the simulator to lock in case of a code which generates infinite simulation loops. I'm not sure if we need that, any thoughts about this?

I have updated the script to use case statement and to run the simulation with -R flag.